mct_quantizers/keras/quantizer_utils.py

```diff
@@ -18,38 +18,38 @@
 
 
 def lut_quantizer(tensor_data: tf.Tensor,
-                  cluster_centers: np.ndarray,
+                  lut_values: np.ndarray,
                   signed: bool,
                   threshold: np.ndarray,
-                  multiplier_n_bits: int,
+                  lut_values_bitwidth: int,
                   eps: float) -> tf.Tensor:
     """
-    Quantize a tensor using a non-uniform quantization based on the pre-defined clusters.
-    1. Scales tensor_data with the threshold into multiplier_n_bits quantization range.
-    2. Assigns cluster centers to each value.
+    Quantize a tensor using a non-uniform quantization based on the pre-defined values.
+    1. Scales tensor_data with the threshold into lut_values_bitwidth quantization range.
+    2. Assigns lut values to each value.
     3. Scales back by multiplying the result by threshold and dividing with the quantization range max value.
     The result is the quantized tensor.
 
     Args:
         tensor_data: Input activation tensor.
-        cluster_centers: the cluster centers to assign the tensor values.
+        lut_values: the values in the look-up table to assign the weights to
         signed: Whether the quantization is signed or not.
         threshold: threshold for quantization.
-        multiplier_n_bits: Number of bits that determines the quantization range
+        lut_values_bitwidth: Number of bits that determines the quantization range
         eps: Small value for numerical stability in division.
```

Comment:
Just a suggestion: 
lut_values_bitwidth: Number of bits to represent the lut values